driver-sync/src/main/com/mongodb/client/internal/Crypt.java

```diff
@@ -273,7 +273,8 @@ private void mark(final MongoCryptContext cryptContext, final String databaseNam
 
     private void fetchKeys(final MongoCryptContext keyBroker) {
         try {
-            for (BsonDocument bsonDocument : keyRetriever.find(keyBroker.getMongoOperation())) {
+            RawBsonDocument mongoOperation = keyBroker.getMongoOperation();
```

Comment:
This seems an unnecessary change.  Can you revert?

Reply:
Will do